### PR TITLE
Front: Gateway: use osrd images pictures

### PR DESCRIPTION
**Pull request by younesschrifi**
Part of #8783

### CODE REVIEW

**Review by SharglutDev — APPROVED** [review]
Lgtm and tested, good job !

**Review by SharglutDev — APPROVED** [review]

**Review by SharglutDev — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -136,6 +136,14 @@ services:
     restart: unless-stopped
     ports: ["4000:4000"]
 
+  osrd-images:
+    image: ghcr.io/openrailassociation/osrd-stable/osrd-images:${TAG_IMAGES-v1.0.0}
```
Comment (flomonster) [inline docker-compose.yml]: It feels okay to take the latest image pushed for the dev setup.

```suggestion
    image: ghcr.io/openrailassociation/osrd-stable/osrd-images:${TAG_IMAGES-latest}
```

**Review by Khoyo — APPROVED** [review]
ops part LGTM

**Review by ghost — APPROVED** [review]
LGTM on devops side, not tested

**Comment by Khoyo** [comment]
You probably want to update `gateway.prod.sample.toml` too (and the charts will need to be updated)

File: front/src/applications/operationalStudies/components/Project/PictureUploader.tsx [hunk 2]
```diff
@@ -85,106 +74,91 @@ function PicturePlaceholder({ image, isValid, tempProjectImage }: PicturePlaceho
...
   setTempProjectImage,
   tempProjectImage,
   safeWord,
   t,
-}: PicturePlaceholderButtonsProps) {
-  async function getRandomImage(url: string) {
+}: PicturePlaceholderButtonsProps) => {
+  const [categories, setCategories] = useState<Categories>({});
+
+  async function getRandomImage(images: string[]) {
```
Comment (flomonster) [inline front/src/applications/operationalStudies/components/Project/PictureUploader.tsx]: I wouldn't have chosen a random image with each click.
The algorithm I have in mind is:
1. When the category is loaded pick a random number
2. When the user click, add one to the previous number (modulo the numbe of images)

This way when you have the same image twice it means that you saw all of them.

**Review by flomonster — COMMENTED** [review]
Thanks for this PR. :pray:

File: front/src/applications/operationalStudies/components/Project/PictureUploader.tsx (hunk 2 above)
Comment (SharglutDev) [reply front/src/applications/operationalStudies/components/Project/PictureUploader.tsx]: So we will always get the same order of images ?

**Pull request merged**